fms/modules/attention.py

```diff
@@ -15,6 +15,57 @@
 from fms.modules.positions import PositionEncoder
 from fms.modules.tp import TPModule
 
+from fms.utils.gptq import GPTQConfig, custom_linear_repr
+
+try:
+    from auto_gptq.utils.import_utils import dynamically_import_QuantLinear
+    IS_AUTOGPTQ_AVAILABLE=True
+except:
+    IS_AUTOGPTQ_AVAILABLE=False
+
+
+def get_linear(
```

Comment:
I'd put this function into either fms/utils/gptq.py or fms/modules/gptq.py. I'm thinking probably the second option, but I'll let @JRosenkranz and @nairbv also chime in

Reply:
in fact, thinking further on compatibility with FP8 Linear layers, we might want to move this into modules/linear.py, a new file just for linear layer handling